src/logic/operate.test.js

```diff
@@ -0,0 +1,36 @@
+import operate from './operate';
+
+test('Add two numbers', () => {
+  const num1 = 2;
+  const num2 = 2;
+  const operation = '+';
+  expect(operate(num1, num2, operation)).toBe('4');
+});
+
+test('Multiply two numbers', () => {
+  const num1 = 2;
+  const num2 = 2;
+  const operation = 'x';
+  expect(operate(num1, num2, operation)).toBe('4');
+});
```

Comment:
- [ ] Good job so far, however, kindly ensure that your tests also cover the following;

 - There should be the usage of Jest snapshots for testing all the components. Here is a[ link](https://jestjs.io/docs/snapshot-testing) to learn how to do this.
 - There should be the usage of React Testing Library to simulate user interaction. This[ link](https://github.com/testing-library/react-testing-library) will help you learn how to write this test.